utils/envvars/parser_test.go

```diff
@@ -132,6 +128,24 @@ func TestEnvVarValidator(t *testing.T) {
 		assert.Equal(t, expectedWarns, gotWarns)
 	})
 
+	t.Run("Parse SAAS host", func(t *testing.T) {
+		userCase := []struct {
+			value   string
+			err     error
+			respVal string
+		}{
+			{value: "host", err: nil, respVal: "host:443"},
+			{value: ":443", err: fmt.Errorf("environment variable %q has invalid format %q. Expected host[:port]", envSaaSHost, ":443"), respVal: ""},
+			{value: "host:443", err: nil, respVal: "host:443"},
```

Comment:
can we use different port in test to check that we don't just set default value

Reply:
Can we also add a few other invalid cases like `host:letters`

Reply:
sure will update tests with different ports, there is no validation for `host:letters` should I add it @BupycHuk ?

Reply:
i thought that `net.SplitHostPort` should be able to validate that, but now I see that it can't. so we can skip it if it's out of scope.